media/css/mozorg/home/home-new.scss

```diff
@@ -99,19 +99,24 @@ header {
 .c-products {
     text-align: center;
 
-    ul {
+    .c-product-columns {
```

Comment:
**Suggestion (design/blocking)**: This updated code isn't built for responsiveness, and was pretty buggy and easy to break when I was testing it out. Instead, I played around with the CSS and got it to work better by enabling `flex-wrap: wrap`. By having all of the cards in one row (esp since the original was `mzp-t-columns-four` and was meant for only 4 items), the responsiveness got pretty finicky. Here are the changes I did on my end to make it look somewhat better.

<kbd>![Screenshot 2024-01-29 at 4 49 42 PM](https://github.com/mozilla/bedrock/assets/42309026/8f1fd00c-3b3e-41be-8985-b238d07cd02f)</kbd>

And the outcome now looks like this:
<kbd>![Screenshot 2024-01-29 at 4 50 20 PM](https://github.com/mozilla/bedrock/assets/42309026/0447bb06-d51c-4fb7-b157-8ceca96f75de)</kbd>

Reply:
Oh goodness, i left this half finished. Thank you for your help stitching this back together